jetty-webapp/src/main/java/org/eclipse/jetty/webapp/StandardDescriptorProcessor.java

```diff
@@ -660,8 +660,8 @@ public void visitSessionConfig(WebAppContext context, Descriptor descriptor, Xml
         if (tNode != null)
         {
             long val = Long.parseLong(tNode.toString(false, true));
```

Comment:
can we change the name of val to something that includes the units.  It is not clear that this is where the fix is.